packages/gamut-labs/src/lib/breakpointHooks/index.ts

```diff
@@ -0,0 +1,43 @@
+import { mapValues } from 'lodash';
+import { breakpoints as originalBreakpoints } from '@codecademy/gamut-styles/utils/variables';
+import { createBreakpoint } from 'react-use';
+
+export type Breakpoint = 'xxs' | 'xs' | 'sm' | 'md' | 'lg' | 'xl';
+export type BreakpointMinWidths = Record<Breakpoint, number>;
+
+export const breakpoints = (mapValues(
+  { xxs: 0, ...originalBreakpoints },
+  (value: string) => parseInt(value, 10)
+) as unknown) as BreakpointMinWidths;
+
+/**
+ * useBreakpoint will provide the current breakpoint to a React component.
+ *
+ * If you think you need this, make sure to think twice.  It is really nice
+ * to allow your styles layer to concern itself about the appearance of a
+ * component at different screen sizes, and not have to maintain two different
+ * components that could drift from eachother.  That being said, sometimes
+ * The content, layout, and/or functionality of a mobile page/component can
```

Comment:
heh, just noticed...

```suggestion
 * the content, layout, and/or functionality of a mobile page/component can
```

Reply:
🧐